packages/web_framework/requirements.txt

```diff
@@ -1 +1,3 @@
 galaxy-data
+galaxy-web-stack
+paste
```

Comment:
The test fails because we are missing the python babel library. Can it be that we need to add it here?